File: MeetMe-Frontend/projects/meetme/src/app/models/schedule-rule.ts
export interface ScheduleRule {
    id:string
    name:string
    ownerId:string,
    timeZoneId:number
    ruleAttributes:ScheduleRuleAttribute[]
}

export interface ScheduleRuleAttribute {
    id:string
    ruleId:string
    type:string
    day:string
    date?:Date
    stepId:number
    from:number
    to:number
}

File: Meetme-Backend/MeetMe.Core/Persistence/Entities/ScheduleRule.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MeetMe.Core.Persistence.Entities
{
    public class ScheduleRule
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public Guid OwnerId { get; set; }
        public int TimeZoneId { get; set; }
        public List<ScheduleRuleAttribute> RuleAttributes { get; set; }
        public ScheduleRule()
        {
            RuleAttributes = new List<ScheduleRuleAttribute>();
        }
    }
}
